static/locales/en-US.yaml

```diff
@@ -145,7 +146,17 @@ Settings:
     Clear Default Instance: Clear Default Instance
     View all Invidious instance information: View all Invidious instance information
     Region for Trending: Region for Trending
-        #! List countries
+    #! List countries
+    External Link Handling:
+      External Link Handling: External Link Handling
+      External Link Handling Tooltip: |
+        External Link Handling: Choose the behavior when a link is clicked.
+        By default FreeTube will open the link in your default browser.
+        (Unless it points to a Youtube Video/Playlist/Channel etc.)
+        You can choose to disable it completely for safety.
+      Open Link: Open Link
+      Open Link After Prompt: Open After After Prompt
```

Comment:
I'm thinking "Open After Prompt" is somewhat unclear and suggests that it will inevitably open the link. I recommend "Ask Every Time" instead.

Reply:
Updated to "Ask Before Opening Link"